packages/react-aria-components/docs/ProgressBar.mdx

```diff
@@ -219,7 +219,8 @@ let c = 2 * r * Math.PI;
 <ProgressBar aria-label="Loading…" value={60}>
   {({percentage}) => <>
     <svg width={64} height={64} viewBox="0 0 32 32" fill="none" strokeWidth={strokeWidth}>
-      <circle cx={center} cy={center} r={r} stroke="var(--circle-color)" />
+      <circle cx={center} cy={center} r={r - (strokeWidth / 2 - .5)} stroke="var(--circle-color)" strokeWidth={1} />
+      <circle cx={center} cy={center} r={r + (strokeWidth / 2 - .5)} stroke="var(--circle-color)" strokeWidth={1} />
```

Comment:
If we do a 1px line I think you can just have one circle?

Reply:
a 1px line? you mean using the path element? or how do you get the empty area in between the two strokes? fill would do center of the circle and a path would still need two elements I think?

Reply:
Previously there were 2 circles, now there are 3. I think we only need 2.

Reply:
ah I think I was confused because it looks like the stroke is 2px even though it says 1px. it's definitely a lot thicker than the progress bar...

Reply:
I don't understand why this works but if you set `strokeWidth` to `0.5` and adjust the radius by `0.25` in either direction it looks right. Why would a .5px svg stroke look like a 1px box shadow?

Reply:
o, you are right, it does look thicker than its prescribed 1px stroke... wonder if it is like that across all the browsers and all screen densities

Reply:
I tested across all three browsers i have, it looks like this gives it 1px. I've pushed the fix, thanks for catching that.